lib/logging/sentry-transport.js

```diff
@@ -42,7 +42,7 @@ module.exports = class SentryTransport extends TransportStream {
       Object.keys(extra).forEach((key) => {
         scope.setExtra(key, extra[key]);
       });
-      if (metadata.hasOwnProperty('response')) {
+      if (!!metadata['response']) {
```

Comment:
Interesting. Can you explain the difference here?

Reply:
@nshki 
```
metadata = {
    response: undefined 
}
```
`metadata.hasOwnProperty('response')` returns `true` for the above object, while `!!metadata['response']` returns `false`.

Reply:
Ah I see, so `hasOwnProperty` doesn't look at the value, it just looks at the presence of a key.

Reply:
Man, nice catch.